### PR TITLE
Setup sentry

### PR DESCRIPTION
**Pull request by rodrei**
@eyko I'm getting a weird behaviour and I'm not being able to figure out why this is happening, this is what I  get:

* When I run the server on the code as is now, when I load the /unsubscribe page I get a `$ Undefined` error. 
* If I s top the server, then comment this line `window.Raven = require('raven-js');`, and then start it again it works. 
* Then, if I uncomment the line `window.Raven = require('raven-js');`, everything works fine.

Basically, the problem is that I can't start the server with the require('raven-js') on the code, If I do that I get an error. But if I start the server without that line, then I can add it without any issues. Maybe it's an issue with require and some weird caching behaviour? I'm lost here.

### CODE REVIEW

File: source/localizable/_head_tags.slim [hunk 1]
```diff
@@ -5,6 +5,7 @@
   title = t('homepage.meta.title')
 meta name="viewport" content="width=device-width, initial-scale=1.0"
 meta charset="utf-8"
+script src="https://cdn.ravenjs.com/3.26.2/raven.min.js" crossorigin="anonymous"
```
Comment (eyko) [inline source/localizable/_head_tags.slim]: Since you're loading Raven here via a script tag, you don't need to do `window.Raven = require('raven-js')` from above (`window.Raven` should be set by the script tag).

File: source/javascripts/homepage.js [hunk 2]
```diff
@@ -17,3 +17,9 @@ window.UnsubscribeForm = require('./sumofus/backbone/unsubscribe_form');
 window.SweetPlaceholder = require('./sumofus/backbone/sweet_placeholder');
 window.FundingTable = require('./sumofus/backbone/funding/funding_table');
 window.ExpensesTable = require('./sumofus/backbone/funding/expenses_table')
+window.Raven = require('raven-js');
```
Comment (eyko) [inline source/javascripts/homepage.js]: The error you're getting is probably because `raven-js` is not in `package.json`. This line should be unnecessary since you've already loaded it in the header via script tag.

Comment (rodrei) [reply source/javascripts/homepage.js]: 🤦‍♂️

File: source/localizable/_head_tags.slim (hunk 1 above)
Comment (rodrei) [reply source/localizable/_head_tags.slim]: Thanks Vince! this has been fixed!
